### DISCUSSION TITLE
Where should I make modifications if I want to adjust the detailed settings of Stochastic RSI?

### DISCUSSION THREAD

**Discussion by sfolaxsel** [conor19w/Binance-Futures-Trading-Bot#38] [General]
![image](https://github.com/conor19w/Binance-Futures-Trading-Bot/assets/134991324/b9d5e96a-086a-4daf-ae5d-00d040f270b9)

I set up the Stochastic RSI chart on Binance as above. 
If I want to use the same settings as shown in the attached picture for a strategy that uses Stochastic, where should I make modifications? 
After modifying it to window=5 and smooth_window=3, 
I compared the fast K & D value printed by the bot with the actual fast K, D value on the chart for that date, and there is a significant difference.

**Comment by conor19w** [comment]
You modify the indicators in tradingStrats.py, theres an error term which crops up mentioned in the readme. With these indicator librarys you need to have a buffer of candles 5 times greater than your largest indicator for accurate results

**Reply by sfolaxsel** [reply]
Thank you for your response. 
However, there is no input field for buffers in the backtester. 
I just want to know if the bot accurately returns the fast K and fast D values at a specific time with my stoch setting(5,3,3)
If there is a difference between the fast K and fast D values at a specific time on the actual Binance chart, it makes it difficult for me to develop a strategy.
